spotbugs/src/test/java/edu/umd/cs/findbugs/ba/generic/TestGenericObjectType.java

```diff
@@ -19,11 +19,7 @@
 
 package edu.umd.cs.findbugs.ba.generic;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
```

Comment:
I think we do not need to use wildcard to import, at least it's not enabled in [Eclipse setting](https://github.com/spotbugs/spotbugs/blob/release-3.1/gradle/.settings-templates/org.eclipse.jdt.ui.prefs).

Reply:
Well, I can switch off that PMD rule, but it means writing a custom PMD rules file...

How would you feel about me introducing AssertJ? Then only one static import is needed.

Reply:
I personally don't care which tool we use.

Reply:
I would avoid adding extra dependencies just to fix some cosmetic issue.

Reply:
@iloveeclipse Fair, although AssertJ is actually quite a nice testing tool in its own right.

But I've written a custom rules file and disabled the static import limitation, anyway.